routes/index.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const router = express.Router();
+
+const productRouter = require('./productRouter');
+
+router.use('/', productRouter.router);
```

Comment:
routes/index.js 파일은 각각의 리소스 라우터로 이동하기 위해 첫 분기처리를 해주는 역할이라고 보시면 됩니다.

따라서 products에 대한 리소스를 가리킬 수 있도록 수정해주세요.
```suggestion
router.use('/products', productRouter.router);
```